src/eureka/S6_planet_spectra/s6_spectra.py

```diff
@@ -681,7 +687,8 @@ def load_s5_saves(meta, log, fit_methods):
     else:
         # No samples for lsq, so just shape it as a single value
         if meta.sharedp:
-            meta = parse_s5_saves(meta, log, fit_methods, 'shared')
+            meta.spectrum_median, meta.spectrum_err = \
+                            parse_s5_saves(meta, log, fit_methods, 'shared')
```

Comment:
This will also need to be reverted now that my PR has been merged into this one

Reply:
Yeah, I noticed this change when reviewing your PR.  Easy enough!